app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -5,62 +5,62 @@ import { Table as NeetoUITable, Avatar, Typography, Dropdown } from "neetoui";
 
 import { monthDateFormatter } from "utils/date";
 
-const Table = ({ contacts = [], onDelete }) => (
-  <div className="notes-table-height w-full">
-    <NeetoUITable
-      allowRowClick
-      rowData={contacts}
-      columnData={[
-        {
-          title: "Name and Role",
-          dataIndex: "firstName",
-          key: "name",
-          width: "30%",
-          render: (firstName, { lastName, role }) => (
-            <div className="flex space-x-2">
-              <Avatar
-                size="large"
-                user={{ name: `${firstName} ${lastName}` }}
-              />
-              <div className="flex flex-col">
-                <Typography style="h4">
-                  {firstName} {lastName}
-                </Typography>
-                <Typography style="body3">{role}</Typography>
-              </div>
-            </div>
-          ),
-        },
-        {
-          title: "Email",
-          dataIndex: "email",
-          key: "email",
-          width: "30%",
-        },
-        {
-          title: "Created at",
-          dataIndex: "createdAt",
-          key: "createdAt",
-          width: "30%",
-          render: monthDateFormatter,
-        },
-        {
-          title: "",
-          dataIndex: "id",
-          key: "id",
-          width: "10%",
-          render: () => (
-            <Dropdown buttonStyle="text" icon={MenuHorizontal}>
-              <div className="p-1.5">
-                <li>Edit</li>
-                <li onClick={onDelete}>Delete</li>
-              </div>
-            </Dropdown>
-          ),
-        },
-      ]}
-    />
-  </div>
-);
+const Table = ({ contacts = [], onDelete }) => {
+  const renderAvatarWithNameAndRole = (firstName, { lastName, role }) => (
+    <div className="flex space-x-2">
+      <Avatar size="large" user={{ name: `${firstName} ${lastName}` }} />
+      <div className="flex flex-col">
+        <Typography style="h4">
+          {firstName} {lastName}
+        </Typography>
+        <Typography style="body3">{role}</Typography>
+      </div>
+    </div>
+  );
+  const renderActionDropdown = () => (
+    <Dropdown buttonStyle="text" icon={MenuHorizontal}>
+      <li>Edit</li>
+      <li onClick={onDelete}>Delete</li>
+    </Dropdown>
+  );
```

Comment:
Move these functions to utils file

Reply:
How should I pass the onDelete function to renderActionDropdown if I move it to utils file?
It is easily accepting the key property from all the column as argument.

Reply:
>How should I pass the onDelete function to renderActionDropdown if I move it to utils file?

@Ishaan-Singhal Please put up a video and show me the different potential ways in which you could pass the args to the function. Then I will point out which one you should go ahead with.

Reply:
<img width="722" alt="Screenshot 2022-09-28 at 10 18 10 PM" src="https://user-images.githubusercontent.com/104645943/192841937-76c6d5c0-70bd-473c-aee0-8b069e77345a.png">

I made render call with an arrow function and restarted the server, then its working fine.

Reply:
>Please put up a video

@Ishaan-Singhal If I am explicitly requesting for something, say a video, then you ought to do exactly that or point out a reason for not doing it. I will review the rest of your PRs, once this task is done.

In the video cover the following points:
- What did you mean by the following statement? `It is easily accepting the key property from all the column as argument.`
- In respect to your question `How should I pass the onDelete function to renderActionDropdown if I move it to utils file?`, show me the different ways by which we can pass arguments to a function, that you know of.

Reply:
https://www.loom.com/share/8f7b0ad8d70644e48b5f350fb8abb960

Check this video.

Reply:
@Ishaan-Singhal https://www.loom.com/share/12c380ea548246109adf72e0f51350d5

Reply:
Thanks for the explanation. This video really helped me. I tried looking to neetoUI docs to understand the purpose and working of the render but I didn't find much there. Here, if I had known about antd and looked into the antd docs in the first place then this problem wouldn't have happened.

Reply:
>I tried looking to neetoUI docs to understand the purpose and working of the render but I didn't find much there. Here, if I had known about antd and looked into the antd docs in the first place then this problem wouldn't have happened.

It's already mentioned in the docs:
<img width="988" alt="image" src="https://user-images.githubusercontent.com/31244999/192973173-a2ddeeef-5319-4435-a5f7-4111cdc99c3e.png">

Reply:
I will add take care of adding more content regarding the `render` key in the docs to avoid further confusion.